app/views/podcast_metrics/_dropdays_card.html.erb

```diff
@@ -0,0 +1,49 @@
+<% chart_id = SecureRandom.uuid %>
```

Comment:
Out of curiousity... why so random?  (And not just `chart_id = "dropdays_card"` or something).